### DISCUSSION TITLE
Is there a reason watermark couldn’t be removed?

### DISCUSSION THREAD

**Discussion by Cdingram** [react-native-maps/react-native-maps#4913] [Q&A]
I might be missing something but if I want to use my own base map with google, I currently set the map type to none and the google watermark still shows up. Is that necessary? Am I still using google services for something that requires the watermark? I’m also assuming there’s no cost involved since I’m not using their basemap with map type set to none correct?

**Comment by salah-ghanim** [comment]
@Cdingram if you see google logo it means you're using google maps, this library doesn't add any artificial UI elements.

see the docs here regarding value of provider.
https://github.com/react-native-maps/react-native-maps/blob/master/docs/mapview.md

and yes for now unfortunately you will have to pay for google if your usage exceed their free tier.
